### DISCUSSION TITLE
RU für Genesenennachweis und Testnachweis

### DISCUSSION THREAD

**Discussion by ruiyan-yu** [Digitaler-Impfnachweis/certification-apis#54] [Q&A] [answered]
Hallo,

Wann können wir mit einer Referenzumgebung für den Genesenennachweis und Testnachweis rechnen?
Werden diese einen andere API Endpoint haben oder wird der aktuelle Endpoint einfach aufgebohrt?

Mit freundlichen Grüßen

Duria e.G

**Comment by oliver-steinbrecher** [comment, marked as answer]
Wir werden zu Ende dieser Woche die Implementierung liefern und an dem bisherigen Endpunkt gemäß API Spezifikation zur Verfügung stellen.

**Reply by dirkexe** [reply]
Ich kann das Stand 8.6.2021 8 Uhr für die RU nicht bestätigen. Bei Abruf eines Genesenennachweis liefert `/api/certify/v2/issue` weiterhin einen Fehler. Hier wird nur gegen das Schema des Impfnachweises geprüft?

**Reply by thinkberg** [reply]
Die Aenderung wird heute auf das Testsystem kommen. Gestern ging das wegen einer Umstellung noch nicht. Eigentlich sollte das schon gestern passieren, wurde aber leider verzoegert.

**Reply by dirkexe** [reply]
Ich konnte jetzt erfolgreich einen Genesenennachweis in der RU abrufen.

**Reply by mge-medisoftware** [reply]
dito, läuft!

**Reply by dirkexe** [reply]
Derzeit versagt der Genesenennachweis in der RU mit:

> Sorry, there is something invalid in your request. Error Validating with Schema #: required key [v] not found
> #/r/0: 3 schema violations found

Hier wird anscheint gegen das falsche Schema (`#/v/0` statt `#/r/0`) geprüft.

**Reply by mge-medisoftware** [reply]
Bei uns funktioniert es. Wir verwenden Content-Type: application/vnd.dgc.v1+json

**Reply by dirkexe** [reply]
Wir verwenden auch `Content-Type: application/vnd.dgc.v1+json`. Und jetzt Stand 09:30 funktioniert es auch wieder. Vielen Dank.
